documentation/OpenAPI/example.rst

```diff
@@ -0,0 +1,187 @@
+    components:
+        schemas:
+            Fact:
+                type: object
+                properties:
+                    _id:
+                        type: string
+                        description: 'Unique ID for the `Fact`'
+                    __v:
+                        type: integer
+                        description: 'Version number of the `Fact`'
+                    user:
+                        type: string
+                        description: 'ID of the `User` who added the `Fact`'
+                    text:
+                        type: string
+                        description: 'The `Fact` itself'
+                    updatedAt:
+                        type: string
+                        description: 'Date in which `Fact` was last modified'
+                    sendDate:
+                        type: string
+                        description: 'If the `Fact` is meant for one time use, this is the date that it is used'
+                    deleted:
+                        type: boolean
+                        description: 'Weather or not the `Fact` has been deleted (Soft deletes are used)'
+                    source:
+                        type: string
+                        description: 'Can be `user` or `api`, indicates who added the fact to the DB'
+                    used:
+                        type: boolean
+                        description: 'Weather or not the `Fact` has been sent by the CatBot. This value is reset each time every `Fact` is used'
+                    type:
+                        type: string
+                        description: 'Type of animal the `Fact` describes (e.g. ‘cat’, ‘dog’, ‘horse’)'
+
+This will allow us to request CatFact API and get a random Fact !
+
+.. _Stoplight: https://stoplight.io/studio/
+.. _CatFacts API: https://alexwohlbruck.github.io/cat-facts/
+
+Jane configuration
+------------------
+
+We need to configure Jane before generation. So we create a ``.jane-openapi`` file::
+
+    <?php
+
+    return [
+        'openapi-file' => __DIR__ . '/schema.yaml',
+        'namespace' => 'CatFacts\Api',
+        'directory' => __DIR__ . '/generated/',
+    ];
+
+It will contains a reference to your main schema file (that file can be linked to other files if you want), the PHP
+namespace you want for generated classes and the directory you want to use.
+
+Jane generation
+---------------
+
+Now we can run generation, basically just require jane with composer (see :doc:`/OpenAPI/introduction` for more details
+about installation) and run the following command:
+
+.. code-block:: bash
+
+    vendor/bin/jane-openapi generate
+
+It will find any ``.jane-openapi`` file and use it as configuration. If your file has a different name, just add
+``-c path/to/my/file`` after the command.
+
+Creating a client
+-----------------
+
+Then you need a Client to bridge between Jane and you application. Jane use HTTPlug_ to make this bridge easier
```

Comment:
```suggestion
Then you need a Client to bridge between Jane and your application. Jane use HTTPlug_ to make this bridge easier
```

> Jane use HTTPlug_

That's not true anymore, is it? cc #120

Reply:
This PR was made before #120, I'll replace everything after this is merged :wink: